asab/web/authz/decorator.py

```diff
@@ -44,15 +44,15 @@ async def wrapper(*args, **kargs):
 				L.error("oauth2_url is not configured ;-(")
 				raise aiohttp.web.HTTPUnauthorized()
 
-			access_token = _get_access_token(request)
+			bearer_token = _get_bearer_token(request)
 
 			# For resistancy against security attacks
-			if access_token is None:
+			if bearer_token is None:
 				raise aiohttp.web.HTTPUnauthorized()
```

Comment:
:crying_cat_face: @ateska @byewokko 
I need the authorization to be optional. This is very strict. It raises 401 error inside asab before I can put some "if" somewhere... I would like the decorator to return None or {} if there is no authorization, not 401. But this might be specific for bsquery and I should implement it there... somehow...

Reply:
I agree, decorator should allow "optional" authorization (or another decorator needs to be added that is "softer" in this manner.

Reply:
juchůůů :pig2:

Reply:
after our discussion, I will only fix the authorization for the "strict" mode. this `optional` approach should be implemented in a separate issue